inventories/README.adoc

```diff
@@ -157,3 +157,100 @@ inventory_satellite/
+You can't really do this with your own list of hosts.
+
+Examples::
+Our first idea could be to define managers and hosts first in an inventory:
++
+.Content of the "bad" `groups_and_hosts` file
+[source,ini]
+----
+include::inventory_loop_hosts/inventory_bad/groups_and_hosts[]
+----
++
+Each manager has a list of hosts, which can look like this:
++
+.List of hosts in `inventory_bad/host_vars/manager_a/provision.yml`
+[source,yaml]
+----
+include::inventory_loop_hosts/inventory_bad/host_vars/manager_a/provision.yml[]
+----
++
+So that we can loop over the list in this way:
++
+.The "bad" way to loop over hosts
+[source,yaml]
+----
+include::inventory_loop_hosts/playbook_bad.yml[]
+----
++
+TIP: check the resulting files using e.g. `head -n-0 /tmp/bad_*`.
++
+As said, no way to limit the hosts provisioned, and no parallelism.
+Compare then with the recommended approach, with a slightly different structure:
++
+.Content of the "good" `groups_and_hosts` file
+[source,ini]
+----
+include::inventory_loop_hosts/inventory_good/groups_and_hosts[]
+----
++
+It is now the hosts and their groups which carry the relevant information, it is not anymore parked in one single list (and can be used for other purposes):
++
+.The "good" variable structure
+----
+$ cat inventory_good/host_vars/host1/provision.yml 
+provision_value: uno
+$ cat inventory_good/group_vars/managed_hosts_a/provision.yml 
+manager_hostname: manager_a
+----
++
+And the provisioning playbook now runs in parallel and can be limited to specific hosts:
++
+.The "good" way to loop over hosts
+[source,yaml]
+----
+include::inventory_loop_hosts/playbook_good.yml[]
+----
++
+The result isn't overwhelming in this simple setup but you would of course better appreciate if the provisioning would take half an hour instead of a fraction of seconds:
++
+.Comparaison of the execution times between good and bad
+----
+$ ANSIBLE_STDOUT_CALLBACK=profile_tasks \
+	ansible-playbook -i inventory_bad playbook_bad.yml 
+Saturday 23 October 2021  13:11:45 +0200 (0:00:00.040)       0:00:00.040 ****** 
+Saturday 23 October 2021  13:11:45 +0200 (0:00:00.858)       0:00:00.899 ****** 
+=============================================================================== 
+create some file to simulate an API call to provision a host ------------ 0.86s
+$ ANSIBLE_STDOUT_CALLBACK=profile_tasks \
+	ansible-playbook -i inventory_good playbook_good.yml 
+Saturday 23 October 2021  13:11:55 +0200 (0:00:00.040)       0:00:00.040 ****** 
+Saturday 23 October 2021  13:11:56 +0200 (0:00:00.569)       0:00:00.610 ****** 
+=============================================================================== 
+create some file to simulate an API call to provision a host ------------ 0.57s
+----
++
+TIP: if for some reason, you can't follow the recommendation, you can at least avoid duplicating too much information by indirectly referencing the hosts' variables as in `"{{ hostvars[item.name]['provision_value'] }}"`. Not so bad...
```

Comment:
```suggestion
TIP: if for some reason, you can't follow the recommendation, you can at least avoid duplicating too much information by indirectly referencing the hosts' variables as in `"{{ hostvars[item.name]['provision_value'] }}"`.
```

Reply:
That was supposed to be a funny reference to the 3rd code example. Not so funny?